io/include/pcl/io/ply/ply.h

```diff
@@ -82,14 +82,14 @@ namespace pcl
         static const char* old_name () { return OLD_NAME; } \
       };
```

Comment:
It might be more relevant to remove the semicolon from here. That way statements in line 85-92 feel more in line with typical C/C++ syntax, in each ever line is terminated by a semicolon. Ping @taketwo for opinion.

Reply:
Good catch. I'll address it in the next commit.

Reply:
Wait for @taketwo 's reply before addressing changes. I already noticed that by following my suggestion we are indeed changing the API and we risk breaking the code for potential users who might use that macro.

Reply:
I don't have a strong opinion here, though I'd rather prefer semicolon-less lines. Reasons: 1) this is in line with how other prominent macros work (think `EIGEN_MAKE_ALIGNED_OPERATOR_NEW`) 2) in the context of class/struct declaration not every line has to end with semicolon, so does not feel unnatural.

Reply:
Comments addressed.

Reply:
Sorry for the confusion. What @taketwo suggested was to proceed with the approach in your original commit. To drop remove your last commit, I suggest you do an interactive rebase.
```
git rebase -i HEAD~2
```
you'll see a list of your last two commits. Drop the last one. Then force push to your remote (assumed here to be named `origin`)
```
git push -f origin fix/remove_trailing_semicolons
```

Reply:
Done.